Assets/Scripts/Skill/MonsoonSkill.cs

```diff
@@ -0,0 +1,63 @@
+using System.Collections;
+using UnityEngine;
+using static GoalManager;
+
+public class MonsoonSkill : Skill
+{
+    private readonly Collider[] targets = new Collider[10];
+
+    private void Start()
+    {
+        skillName = "Monsoon";
+        requireMana = 4;
+        damage = 10;
+        radius = 3f;
+        effectToSpawn = Managers.Resource.Load<GameObject>($"Prefabs/Skill/Monsoon");
+    }
```

Comment:
```suggestion
    public override void Init()
    {
        SkillName = "Monsoon";
        RequireMana = 4;
        Damage = 10;
        Radius = 3f;
        base.Init();
    }
```
스킬 클래스 구조가 변경되면 이 내용으로 적용해주세요.